org.compiere.db.oracle.provider/src/org/compiere/db/DB_Oracle.java

```diff
@@ -319,12 +313,14 @@ public String toString()
         StringBuilder sb = new StringBuilder("DB_Oracle[");
         sb.append(m_connectionURL);
         try
-        {
-            StringBuilder logBuffer = new StringBuilder(50);
-            logBuffer.append("# Connections: ").append(m_ds.getNumConnections());
-            logBuffer.append(" , # Busy Connections: ").append(m_ds.getNumBusyConnections());
-            logBuffer.append(" , # Idle Connections: ").append(m_ds.getNumIdleConnections());
-            logBuffer.append(" , # Orphaned Connections: ").append(m_ds.getNumUnclosedOrphanedConnections());
+        {        	
+            StringBuilder logBuffer = new StringBuilder();
+            HikariPoolMXBean mxBean = m_ds.getHikariPoolMXBean();            
+
+            logBuffer.append("# Connections: ").append(mxBean.getTotalConnections());
+            logBuffer.append(" , # Busy Connections: ").append(mxBean.getActiveConnections());
+            logBuffer.append(" , # Idle Connections: ").append(mxBean.getActiveConnections());

```

Comment:
why getActiveConnections() for both busy and idle ?

Reply:
Good catch! Copy paste error. I resolved it (in multiple places).

Reply:
seems the fix is gone in latest push